### PR TITLE
REACH-157 NodeView for Watch node

### PR DESCRIPTION
**Pull request by pbidenko**
Use WatchNodeView for watch node.
To work correctly, this pull request requires https://github.com/Benglin/Dynamo/pull/255.

### CODE REVIEW

File: app/scripts/views/NodeViews/Watch.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ define(['backbone', 'underscore', 'jquery', 'BaseNodeView'], function(Backbone,
 
     renderNode: function(){
 
-    	var pretty = this.model.get('lastValue') != undefined ? JSON.stringify(this.model.get('lastValue'), this.prettyPrint, 2) : this.model.get('lastValue');
+    	var pretty = this.model.get('lastValue') != undefined ? JSON.parse(JSON.stringify(this.model.get('lastValue'), this.prettyPrint, 2)) : this.model.get('lastValue');
```
Comment (pbidenko) [inline app/scripts/views/NodeViews/Watch.js]: Parse stringified value to remove unneeded quotes.

File: app/scripts/views/NodeViews/Watch.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ define(['backbone', 'underscore', 'jquery', 'BaseNodeView'], function(Backbone,
     	}
 
     	if (typeof val === "string"){
-    		return val.replace(new RegExp("\t", 'g'), "").replace(new RegExp("\n", 'g'), "<br>")
+    		return val.replace(new RegExp("\t", 'g'), "&#09;").replace(new RegExp("\n", 'g'), "<br>");
```
Comment (pbidenko) [inline app/scripts/views/NodeViews/Watch.js]: Is there any reason why we should completely remove tabulations?

File: app/styles/main.css [hunk 3]
```diff
@@ -1305,11 +1305,16 @@ input[type="button"].btn-block {
     border: 1px solid #BBB;
 }
 
-    .watch-data-container pre {
-        padding: 0;
-        margin: 0;
-        text-align: left;
-    }
+.watch-data-container pre {
+    -moz-tab-size: 4;
```
Comment (pbidenko) [inline app/styles/main.css]: Set tab size to 4 (8 by default)

**Comment by pbidenko** [comment]
Hi @pboyer,
Please, do not merge this pull request until we get clear with https://github.com/Benglin/Dynamo/pull/255#discussion_r22651432

**Comment by pboyer** [comment]
Status update? Looks like you're unifying 255 and 256?

**Comment by pbidenko** [comment]
Yes, Peter, we are going to unify 255 and 256.

**Comment by pboyer** [comment]
Hi @pbidenko, 

Can you bring this up to date?

**Comment by pboyer** [comment]
ping @pbidenko

**Comment by pbidenko** [comment]
Hey @pboyer , thanks for the reminder :)

**Comment by pbidenko** [comment]
To work correctly, this pull request requires https://github.com/DynamoDS/Dynamo/pull/3730

**Pull request closed without merging**